ARKitExample/README.md

```diff
@@ -6,7 +6,7 @@ The project demonstrates how to use Twilio's Programmable Video SDK to stream an
 
 See the master [README](https://github.com/twilio/video-quickstart-ios/blob/master/README.md) for instructions on how to generate access tokens and connect to a Room.
 
-This example requires Xcode 10.0, and the iOS 12.0 SDK. An iOS device with an A9 CPU or greater is needed for ARKit to function properly.
+This example requires Xcode 11.0, and the iOS 12.0 SDK. An iOS device with an A9 CPU or greater is needed for ARKit to function properly.
```

Comment:
This is accurate, but is it strange that we require Xcode 11 which does not include the iOS 12 SDK?

Reply:
This is true, but say a developer wanted to ship this quickstart to the app store, they would require Xcode 11 to do so. My thought here was saying that we recommend the use of Xcode 11 for all new development using the QuickStarts? I wouldn't want to say that it requires Xcode 10. So if we were to say anything, would we just say we require iOS 12.0 SDK and leave the Xcode version off?